doc/porting/board_porting.rst

```diff
@@ -258,3 +259,111 @@ symbols.
 
 Having fixed settings be user-configurable might be confusing, and would allow
 the user to create broken configurations.
+
+Kconfig extensions
+==================
+
+Zephyr uses the `Kconfiglib <https://github.com/ulfalizer/Kconfiglib>`_ Kconfig
+implementation, which adds the following extensions to the `Kconfig language
+<https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt>`_:
+
+- The ``gsource`` statement, which includes each file matching a given wildcard
+  pattern.
+
+  Consider the following example:
+
+  .. code-block:: none
+
+      gsource "foo/bar/*/Kconfig"
+
+  If the pattern ``foo/bar/*/Kconfig`` matches the files
+  :file:`foo/bar/baz/Kconfig` and :file:`foo/bar/qaz/Kconfig`, the statement
+  above is equivalent to the following two ``source`` statements:
+
+  .. code-block:: none
+
+      source "foo/bar/baz/Kconfig"
+      source "foo/bar/qaz/Kconfig"
+
+  .. note
+
+      The wildcard patterns accepted are the same as for the Python `glob
+      <https://docs.python.org/3/library/glob.html>`_ module.
+
+  If no files match the pattern, ``gsource`` has no effect. This means that
+  ``gsource`` also functions as an "optional" include statement (similar to
+  ``-include`` in Make):
+
+  .. code-block:: none
+
+      gsource "foo/include-if-exists"
+
+  .. note::
+
+     Wildcard patterns that do not include any wildcards (e.g. ``*``) only
```

Comment:
clearer would be:

    ... any wildcard symbols (e.g., ``*``)

Reply:
Will change